### ISSUE TITLE
Academic resume theme

### ISSUE THREAD

**Issue by tlienart** [tlienart/FranklinTemplates.jl#116]
--> https://github.com/tlienart/ft-academic-resume

**Comment by Pramodh-G** [comment]
Hey @tlienart, thanks for working on this. 

Could you please give an ETA for this theme? This is the only thing preventing me from switching from hugo to Franklin.jl

**Comment by tlienart** [comment]
Hello @Pramodh-G it's virtually already ready, I just need help finishing it up, so if you're willing to try it and ping me from where you try to do the last few adjustments, that would be great and would motivate me in pushing this over the line :-)

If you're willing to try, you should just fork https://github.com/tlienart/ft-academic-resume and start filling your content. Ideally you'd give me collaborator access to your repo so that I can more easily help when there are things that are not polished.

Would that be of interest?

**Comment by Pramodh-G** [comment]
Hey @tlienart, sorry for the late reply, was caught up in something. I am not well versed in CSS or HTML / Web Development in general, which is why I'm looking for static site generators; Could you please explain what I would be doing if I were to try to adjust the last few things? Can it be done without knowing CSS/HTML

**Comment by tlienart** [comment]
Yes definitely; effectively what I'm suggesting is that you try to use it, fill it with your content, and ping me whenever there's something you don't like or that doesn't work; I'll help out (be it with CSS or whatever). But if you drive it by trying to get the website done, it'll motivate me to fix the inevitable small flaws left or missing parts 😄

**Comment by Pramodh-G** [comment]
Sure, I'll try to do it over the weekend and get back to you; Can we take this discussion to slack?

**Comment by tlienart** [comment]
Ideally you'd fork the repo and ping me from there, feel free to also reach out on Slack but it's far easier for me to take tasks from GitHub and do it async

**Comment by manuelhaussmann** [comment]
Hi all, what is the current status here? Judging from your nice website Parmodh it looks pretty useable to me.

**Comment by tlienart** [comment]
@manuelhaussmann yes the template is essentially usable, you can clone the repo linked to above and start from that. It's not part of the official templates yet as I'd like more people to test it and help me fix the smaller remaining issues before doing so 😀

**Comment by manuelhaussmann** [comment]
That sounds like a plan! (Btw. thanks for all your work on Franklin et al.)
